coiote-aws-iot-cloud-formation/cloudFormation.json

```diff
@@ -0,0 +1,251 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Resources": {
+    "IamForLambda": {
```

Comment:
Check this https://stackoverflow.com/questions/57660653/how-to-create-some-random-or-unique-value-in-a-cloudformation-template
to avoid error with "already exists".

Reply:
If we don't provide the "RoleName" attribute the created role's name will be: `<stack-name>-<resource-name>-<unique-random-string>`. I'll use that.